src/Rakaposhi.Core/Rakaposhi.Business.Core/DBRepository/RoleRepository.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Rakaposhi.Business.Core.BaseRepository;
+using Rakaposhi.Business.Core.DataObjects;
+using Rakaposhi.Data;
+using System.Data.Common;
+using System.Text.Json;
+
+namespace Rakaposhi.Business.Core.DBRepository
+{
+    public class RoleRepository : IRoleRepository
+    {
+        private IDatabase _db;
+
+        public RoleRepository()
+        {
+            _db = Database.Instance;
+        }
+
+        public void Add(Role entity)
+        {
+            DbCommand cmd = _db.CreateStoreProcedure(storeProcedureName: StoreProcds.INSERT);
+            _db.AddParameter(cmd, paramName: Params.NAME, entity.Name);
+            _db.AddParameter(cmd, paramName: Params.DESCRIPTION, entity.Description);
```

Comment:
you missed second param caption for both params. Like you did for first one paramName

Reply:
fixed